x/clp/keeper/calculations.go

```diff
@@ -78,37 +53,19 @@ func CalcSwapPmtp(toRowan bool, y, pmtpCurrentRunningRate sdk.Dec) sdk.Dec {
 	return y.Mul(sdk.NewDec(1).Add(pmtpCurrentRunningRate))
 }
 
-func SetInputs(sentAmount sdk.Uint, to types.Asset, pool types.Pool) (sdk.Uint, sdk.Uint, sdk.Uint, bool) {
-	var X sdk.Uint
-	var Y sdk.Uint
-	var x sdk.Uint
-	toRowan := true
-	if to == types.GetSettlementAsset() {
-		Y = pool.NativeAssetBalance
-		X = pool.ExternalAssetBalance
-	} else {
-		X = pool.NativeAssetBalance
-		Y = pool.ExternalAssetBalance
-		toRowan = false
-	}
-	x = sentAmount
-
-	return X, x, Y, toRowan
-}
-
 func GetSwapFee(sentAmount sdk.Uint,
 	to types.Asset,
 	pool types.Pool,
-	normalizationFactor sdk.Dec,
-	adjustExternalToken bool,
-	pmtpCurrentRunningRate sdk.Dec) sdk.Uint {
-	X, x, Y, toRowan := SetInputs(sentAmount, to, pool)
-	swapResult, err := CalcSwapResult(toRowan, normalizationFactor, adjustExternalToken, X, x, Y, pmtpCurrentRunningRate)
-	if err != nil {
-		// this branch will never be reached as err will always be nil
-		return sdk.Uint{}
+	pmtpCurrentRunningRate sdk.Dec,
+	marginEnabled bool) sdk.Uint {
```

Comment:
@AustinoBombino similarly here we should move `GetSwapFee` to the margin feature toggle file as the signature of the function changes and preserve the previous one in none feature toggle file, you can see more details on how to implement that in my previous comment.